views/item/new.ejs

```diff
@@ -1,35 +1,99 @@
 <%- contentFor('title') %> EcoFoodie
 <%- contentFor('head') %>
+
+<link rel="stylesheet" type="text/css" href="/stylesheets/itemNew.css">
 <script type="application/javascript" src="/scripts/uploader.js"></script>
 <%- contentFor('body') %>
-<h1>New Items</h1>
+
+
+
+<img src="/images/greenArrow.png" alt="green arrow" style="width:128px;height:128px;">
+<img src="/images/EcoFoodieLOGO.png" alt="Eco Foodie Logo" style="width:128px;height:128px;">
+
 
 <form method="post" action="/item">
   <div id="other" class="form-group" style="display: none;">
     <input type="text" class="form-control" name="other" value="" placeholder="Please describe...">
   </div>
 
-  <div class="form-group">
+<!--  <div class="form-group">
     <label>Restaurant</label>
     <textarea class="form-control" rows="2" name="restaurantId" ></textarea>
-  </div>
+  </div> -->
 
   <div class="form-group">
-    <label>Picture</label>
+    <h3>Picture</h3>
     <input type="file" class="form-control" data-name="photo" />
   </div>
 
-  <div class="form-group">
-    <label>Bin</label>
-    <input type="textarea" class="form-control" name="bin" value="" />
+
+  <h2>What is this photo about?</h2>
+  <br>
+
+  <select id="category">
+  <option value="Plastic">Plastic</option>
+  <option value="LandFill">LandFill</option>
+  <option value="Recycle">Recycle</option>
+  </select>
+  <br>
+  <br>
+
+    <h1>Bin</h1>
+
+
+    <br>
+
+    <div class="radio-toolbar">
+        <input type="radio" id="radioApple" name="radioFruit" value="apple" checked>
+        <label for="radioApple">Recycling   <div class="left-img1"></div></label>
+        <br>
+        <br>
+    </div>
+
+
+    <div class="radio-toolbar2">
+        <input type="radio" id="radioBanana" name="radioFruit" value="banana">
+        <label for="radioBanana">Compost <div class="left-img2"></div></label>
+        <br>
+        <br>
+
+
+    <div class="radio-toolbar3">
+
+        <input type="radio" id="radioOrange" name="radioFruit" value="orange">
+        <label for="radioOrange">  Trash <div class="left-img3"> </div></label>
+        <br>
+        <br>
+
+
+
+    <p>&nbsp;</p>
+
+
+
+
+  <br>
+
+
+
+
+
+
+
+  <!-- <input type="textarea" class="form-control" name="bin" value="" />
   </div>
 
   <div class="form-group">
     <label>Category</label>
     <input type="number" class="form-control" name="categoryId" value="" />
-  </div>
+  </div> -->
+
+
+
+
+
 
   <div class="form-group">
-    <button class="btn btn-primary btn-block" type="submit">Submit</button>
+    <a href="#" class="button">Next</a>
```

Comment:
Instead of replacing the original Submit button tag, can you just edit it so that it uses your new styling and has the word "Next"? 

This new button doesn't do anything when you press it, whereas the original button tag would submit the form on the page.